notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>For step 3, are you performing three steps? 1. Install virtualenv, 2. Create a virtual enironmnet, and 3. Activate the virtual environment. If so, I would recommend using:</p><p> </p><p>3.Set up a virtual environment:</p><p>   a. Install virtualenv.</p><p>   b. Create a virtual enironmnet.</p><p>   c. Activate the virtual environment.</p><p> </p><p>If there are links to the resources for a, b, and c, I would add those as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>